### PR TITLE
removed files 10-dhcp-config & 99-default, because they are not neede…

### PR DESCRIPTION
**Pull request by kobi86**
…d, they are being created in the custome-config file

Some order in the Repo :)

### CODE REVIEW

**Comment by kobi86** [comment]
@phoracek this is what you wanted ? PR that will include some cleanup and the fix? 
or you want to do it with two different PRs?

File: custom-config.fcc [hunk 1]
```diff
@@ -132,11 +118,8 @@ systemd:
...
         ConditionKernelCommandLine=custom-config
-
         [Service]
         Type=oneshot
         ExecStart=/usr/local/bin/create-datastore
-
         [Install]
-        WantedBy=coreos-installer.target
-    
+        WantedBy=coreos-installer.target
```
Comment (phoracek) [inline custom-config.fcc:125]: This is missing the last EOL

File: 10-dhcp-config.conf [hunk 2]
```diff
@@ -1,2 +0,0 @@
-[main]
```
Comment (phoracek) [inline 10-dhcp-config.conf:1]: Please format the commit message. The title line should be under 70 characters. The rest should be divided in a paragraph bellow. You can see it here that even GitHub gets confused by the long line and it does not help readability of the log.

You say that these files just get created. Is there any guidance on how to create them? Is it just a regular process while working with RHCOS and we assume folks know how to generate it?

File: custom-config.fcc [hunk 3]
```diff
@@ -114,14 +102,12 @@ systemd:
       contents: |
         [Unit]
         Description=Capture MAC address from kargs
-        After=create-datastore.service
+        After=coreos-installer.service
```
Comment (phoracek) [inline custom-config.fcc:105]: Would you please comment on this change in the commit message?

**Review by phoracek — CHANGES_REQUESTED** [review]
Some nits, but overall looks well. Commented on diffs.

By splitting to commits what I mean was - if you are moving and changing a file, move it in one commit and change in another to allow for a clear diff.

I won't fight against redundant file cleanup in this PR. As long as it does not bring risk of regressions and does not make the actual bugfix harder to review.

**Pull request closed without merging**